<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="play_btn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="end_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>35</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Refresh">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>170</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>All Songs</string>
    </property>
   </widget>
   <widget class="QSlider" name="Progressbar">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>501</width>
      <height>20</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="volumeslider">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playing_song">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>361</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>__-__-__-__-__</string>
    </property>
   </widget>
   <widget class="QLabel" name="volume_label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>240</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_label">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QPushButton" name="modebtn">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>90</y>
      <width>45</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>MODE</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelabel">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>91</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="SearchLine">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="playlistwidget">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>330</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addplaylist_btn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>300</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="playlist_label">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Playlists</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addtoPlaylist">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>270</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="music_label">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>2</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sound_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>240</y>
      <width>37</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCheckBox" name="sound3d">
    <property name="geometry">
     <rect>
      <x>571</x>
      <y>141</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delSong">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>310</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="delPlaylist">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>330</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="searchicon">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>225</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="sound3dlabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>130</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
